backend/smody/src/test/java/com/woowacourse/smody/member/service/MemberServiceTest.java

```diff
@@ -178,7 +181,197 @@ void updateMember_multiThread() throws InterruptedException {
+                    () -> assertThat(mentionResponse.get(0).getNickname()).isEqualTo("조조그린"),
+                    () -> assertThat(mentionResponse.get(1).getNickname()).isEqualTo("더즈"),
+                    () -> assertThat(mentionResponse.get(2).getNickname()).isEqualTo("토닉"),
+                    () -> assertThat(mentionResponse.get(3).getNickname()).isEqualTo("알파"),
+                    () -> assertThat(mentionResponse.get(4).getNickname()).isEqualTo("조그린"),
+                    () -> assertThat(mentionResponse.get(5).getNickname()).isEqualTo("그랑조"),
+                    () -> assertThat(mentionResponse.get(6).getNickname()).isEqualTo("조"),
+                    () -> assertThat(mentionResponse.get(7).getNickname()).isEqualTo("양조장")
+            );
+        }
+
+        @DisplayName("글자가 없고 커서 ID도 없고 크기가 5일때")
+        @Test
+        void mentionAll_sizeFive() {
+            // given
+            PagingParams pagingParams = new PagingParams(null, 5);
+            fixture.회원_추가("조그린", "a@naver.com");
+            fixture.회원_추가("그랑조", "b@naver.com");
+            fixture.회원_추가("조", "c@naver.com");
+            fixture.회원_추가("양조장", "d@naver.com");
+
+            // when
+            List<MentionResponse> mentionResponse = memberService.mentionTo(pagingParams);
+
+            // then
+            assertAll(
+                    () -> assertThat(mentionResponse.size()).isEqualTo(5),
+                    () -> assertThat(mentionResponse.get(0).getNickname()).isEqualTo("조조그린"),
+                    () -> assertThat(mentionResponse.get(1).getNickname()).isEqualTo("더즈"),
+                    () -> assertThat(mentionResponse.get(2).getNickname()).isEqualTo("토닉"),
+                    () -> assertThat(mentionResponse.get(3).getNickname()).isEqualTo("알파"),
+                    () -> assertThat(mentionResponse.get(4).getNickname()).isEqualTo("조그린")
+            );
+        }
+
+        @DisplayName("글자가 없고 커서 ID는 있고 크기가 10일 때")
+        @Test
+        void mentionAll_existCursorId() {
+            // given
+            PagingParams pagingParams = new PagingParams(null, 0, 알파_ID);
+            fixture.회원_추가("조그린", "a@naver.com");
+            fixture.회원_추가("그랑조", "b@naver.com");
+            fixture.회원_추가("조", "c@naver.com");
+            fixture.회원_추가("양조장", "d@naver.com");
+
+            // when
+            List<MentionResponse> mentionResponse = memberService.mentionTo(pagingParams);
+
+            // then
+            assertAll(
+                    () -> assertThat(mentionResponse.size()).isEqualTo(4),
+                    () -> assertThat(mentionResponse.get(0).getNickname()).isEqualTo("조그린"),
+                    () -> assertThat(mentionResponse.get(1).getNickname()).isEqualTo("그랑조"),
+                    () -> assertThat(mentionResponse.get(2).getNickname()).isEqualTo("조"),
+                    () -> assertThat(mentionResponse.get(3).getNickname()).isEqualTo("양조장")
+            );
```

Comment:
```suggestion
            assertAll(
                    () -> assertThat(mentionResponse.size()).isEqualTo(4),
                    () -> assertThat(mentionResponse).map(MentionResponse::getNickname)
                            .containsExactly("조그린", "그랑조", "조", "양조장")
            );
```
위 처럼 작성할 수 있습니다.  다른 테스트에도 적용할 수 있을 것 같아요

Reply:
수정하였습니다